src/commands/createNewProject/initProjectForVSCode.ts

```diff
@@ -124,6 +128,20 @@ async function writeExtensionRecommendations(projectCreator: ProjectCreatorBase,
     );
 }
 
+function addToFilesExcludeSetting(filesToExclude: string | string[], data: {}): { [key: string]: boolean } {
+    // tslint:disable-next-line:no-unsafe-any
```

Comment:
Yeah because data is type `{}` so `data[filesExcludeSetting]` has type any. Technically the user could have invalid settings where `data[filesExcludeSetting]` was the wrong type, but I'm not too worried about it.